netboot-oci-artifacts.md

```diff
@@ -49,127 +71,69 @@ For the rest of image definition refer to the [OCI specs](https://github.com/ope
-      "size": 102417772,
+      "mediaType": "application/x-netboot-file+zstd",
+      "digest": "sha256:8526a40f8b5aa92dd35b01c03f2e748912d56b84c8675fe2f21e36a39b8eb388",
+      "size": 565332,
       "annotations": {
-        "org.opencontainers.image.description": "Anaconda installer init RAM disk",
-        "org.opencontainers.image.title": "initrd.img"
+        "org.opencontainers.image.title": "grubx64.efi",
+        "org.pulpproject.netboot.src.digest": "sha256:735284626212a6267c0e90dab2428e8f82c182af17aec567c80838d219d9fa42",
+        "org.pulpproject.netboot.src.size": "2532984"
       }
     },
     {
-      "mediaType": "application/x-netboot-file",
-      "digest": "sha256:32e77976ebbc915f77dd7f15d66a52cb177d5a9d2ee1794b173390b67495c047",
-      "size": 946736,
+      "mediaType": "application/x-netboot-file+zstd",
+      "digest": "sha256:d50baa5d4bf3af0fabebc7871b884be83f368c83dbe0ce1733087615808a6c15",
+      "size": 41637,
       "annotations": {
-        "org.opencontainers.image.description": "SecureBoot shim EFI binary",
-        "org.opencontainers.image.title": "shim.efi"
+        "org.opencontainers.image.title": "pxelinux.0",
+        "org.pulpproject.netboot.src.digest": "sha256:dfcdf626efa753db88de0bf513c4c2e1c4e46cf084371e294e5f6864f16c2e01",
+        "org.pulpproject.netboot.src.size": "42555"
       }
     },
     {
-      "mediaType": "application/x-netboot-file",
-      "digest": "sha256:735284626212a6267c0e90dab2428e8f82c182af17aec567c80838d219d9fa42",
-      "size": 2532984,
-      "annotations": {
-        "org.opencontainers.image.description": "Grub bootloader binary",
-        "org.opencontainers.image.title": "grubx64.efi"
-      }
-    }
-  ],
-  "annotations": {
-    "org.opencontainers.image.created": "2024-02-06T17:18:04Z",
-    "org.opencontainers.image.description": "Anaconda installer bootloader and Linux kernel/initrd. Usage: oras pull ghcr.io/lzap/bootc-netboot-example:TAG"
-  }
-}
-```
-
-
-## Image manifest tag naming conventions
-
-An image manifest containing Netboot files MUST be tagged in the format of ``name-version-architecture`` where:
-
-* ``name`` SHOULD be OS lowercase name ( e.g rhel, fedora)
-* ``version`` SHOULD specify OS version number in lowercase alphanumeric with dots or underscores without
-  dash character (e.g. 39 or 2024.04 or 9.2).
-  For rolling-release operating systems, version SHOULD contain date or timestamp, sortable order
-  is preferred.
-* ``architecture`` SHOULD be one of the values listed in the Go Language document for
-  [GOARCH](https://go.dev/doc/install/source#environment).
-
-
-## Image index containing OCI artifacts
-
-For the multi-arch builds it is possible to create [OCI image index](https://github.com/opencontainers/image-spec/blob/main/image-index.md)
-that points to specific image manifests, ideal for one or more platforms.
-
-Every entry of image manifest MAY specify additional OPTIONAL metadata via annotations.
-
-
-## Example OCI Image Index
```

Comment:
please bring back the example of image index

Reply:
Sure, I did not see what is different from the other example.

Reply:
https://github.com/pulp/netboot-oci-specs/pull/4/files#diff-3fececa299f23a3ee27a30b1d37c6e4ba65ab03f7904d1842ad958dc001fe33eL133

Reply:
oh i totally missed this sure